server/routes/creatingLicences/handlers/additionalLicenceConditionUploadsHandler.ts

```diff
@@ -16,13 +17,13 @@ export default class AdditionalLicenceConditionUploadsHandler {
     const { user } = res.locals
     const licence = await this.licenceService.getLicence(licenceId, user)
 
-    const conditions = licence.additionalLicenceConditions.filter(c => c.code === conditionCode)
+    const conditions = licence.additionalLicenceConditions.filter((c: AdditionalCondition) => c.code === conditionCode)
```

Comment:
is this just a stylistic thing? typescript should be able to infer the types anyway?

Reply:
I was getting type errors within IntelliJ when missing the cast. It seems at bit fickle within the IDE to be fair.